code/CO2_monitor.ino

```diff
@@ -154,9 +169,21 @@ SCD30 airSensor;
   float humidity_web    = 0.0;
   float co2_web         = 0.0;
 
-  // create AsyncWebServer object on port 80 (port 80 for http)
-  AsyncWebServer server(80);
+  #if WIFI_WEBSERVER
+    // create AsyncWebServer object on port 80 (port 80 for http)
+    AsyncWebServer server(80);
+  #endif
+  #if WIFI_MQTT
+    WiFiClient espClient;
+    PubSubClient client(espClient);
```

Comment:
Personally, I would prefer to call the object `mqttClient` or something similar (would make it easier to immediately recognize what this is further down in the code), i.e. change it to
`PubSubClient mqttClient(espClient);`

What is your opinion on that?